packages/aws-cdk-lib/aws-efs/lib/efs-file-system.ts

```diff
@@ -350,6 +357,188 @@ export interface FileSystemAttributes {
   readonly fileSystemArn?: string;
 }
 
+/**
+ * Properties for the ReplicationConfiguration.
+ */
+export interface ReplicationConfigurationProps {
+  /**
+   * The existing destination file system for the replication.
+   *
+   * @default - create a new file system for the replication
```

Comment:
```suggestion
```

Reply:
@gracelu0 
I believe the @default description is essential because the `destinationFileSystem` is optional. Removing this description could cause the CI to fail.

What do you think?

Reply:
Ohh right, thanks for pointing that out! perhaps “None” or “No destinationFileSystem set” instead then?

Reply:
Thanks! I've updated it.